pokebattle/settings/production.py

```diff
@@ -21,7 +21,7 @@
 MEDIA_ROOT = base_dir_join('mediafiles')
 MEDIA_URL = '/media/'
 
-SERVER_EMAIL = 'pokebattlesender@gmail.com'
+SERVER_EMAIL = 'pamella@vinta.com.br'
```

Comment:
This would be preferably a setting: 
`SERVER_EMAIL = config('SERVER_EMAIL', default='')`

Reply:
Done!